### ISSUE TITLE
Add Gifting To Site

### ISSUE THREAD

**Issue by agentgamer100** [Minehut/Meta#892]
### Feature request

**Checklist**
- [X] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [X] This is not a bug report, support request or plugin request
- [X] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [X] This issue does not contain any sensitive information 

**Platform**
Website

**Describe the feature**
Add gifting credits to other minehut accounts. Say if someone wanted to give someone 800 credits, they could click a button to find an account linked with a email, make their purchase, and credits are given to them. I've left some quick artwork to show how I invision it:
https://ibb.co/SJ2VjGR
https://ibb.co/6nzwPZd

**How the feature is useful**
Basically right now the only way to give someone credits is for them to give you their account. This is completly unsafe. A sure way for someone to give you credits without giving someone your information is by gifting them credits, so their account and servers are safe.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by Blocksnmore** [comment]
Dupe of #377
